README.md

```diff
@@ -1,98 +1,78 @@
-
-## Installation
-
-Detailed instructions on how to install, configure, and get the project running.
-This should be frequently tested to ensure reliability. Alternatively, link to
-a separate [INSTALL](INSTALL.md) document.
-
-## Configuration
-
-If the software is configurable, describe it in detail, either here or in other documentation to which you link.
-
-## Usage
-
-Show users how to use the software.
-Be specific.
-Use appropriate formatting when showing code snippets.
-
-## How to test the software
-
-If the software includes automated tests, detail how to run those tests.
-
-## Known issues
-
-Document any known significant shortcomings with the software.
-
-## Getting help
-
-Instruct users how to get help with this software; this might include links to an issue tracker, wiki, mailing list, etc.
-
-**Example**
-
-If you have questions, concerns, bug reports, etc, please file an issue in this repository's Issue Tracker.
-
-## Getting involved
-
-This section should detail why people should get involved and describe key areas you are
-currently focusing on; e.g., trying to get feedback on features, fixing certain bugs, building
-important pieces, etc.
-
-General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](CONTRIBUTING.md).
-
-
-----
-
-## Open source licensing info
-1. [TERMS](TERMS.md)
-2. [LICENSE](LICENSE)
-3. [CFPB Source Code Policy](https://github.com/cfpb/source-code-policy/)
-
-
-----
-
-## Credits and references
-
-1. Projects that inspired you
-2. Related projects
-3. Books, papers, talks, or other sources that have meaniginful impact or influence on this project
+jenkins-automation
+==================
+
+Repos automated by Jenkins DSL 
+
+This repository contains the automation scripts for dev Jenkins.
+We use [Job DSL](https://github.com/jenkinsci/job-dsl-plugin/wiki) to automate.
+The `DSL Master` job will watch all listed repositories for changes, and rebuild all jobs defined in those repositories.
+To create Jenkins jobs for your project, you must create a `jenkins.groovy` file at the root of your project's primary repository.
+Then add your project's primary repo to the list of watched repos in your organization's `jenkins-automation-private` repository.
+
+You should create your jobs using the templates defined in 
+[jenkins.groovy](https://github.com/cfpb/jenkins-automation/blob/master/jenkins.groovy).
+
+
+## Installation 
+
+### Jenkins Plugins
+
+- https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin
+- https://wiki.jenkins-ci.org/display/JENKINS/Build+Flow+Plugin
+- https://wiki.jenkins-ci.org/display/JENKINS/EnvInject+Plugin
+- http://wiki.jenkins-ci.org/display/JENKINS/Git+Plugin
+- http://wiki.jenkins-ci.org/display/JENKINS/Multiple+SCMs+Plugin
+- http://wiki.jenkins-ci.org/display/JENKINS/NodeJS+Plugin
+- http://wiki.jenkins-ci.org/display/JENKINS/Clone+Workspace+SCM+Plugin
+
+### Job DSL Bootstrap
```

Comment:
I went cross-eyed reading the Bootstrap procedure.  I think the biggest reason is you're talking about bootstrapping to create stuff but I don't know what the stuff is.  What is bootstrap creating?  Also, how do I copy this sample code into a jenkins job?